Documentation/guides/building-apps/build-items.md

```diff
@@ -84,6 +101,25 @@ Files with a Build action of `AndroidJavaLibrary` are Java
 Archives ( `.jar` files) that will be included in the final Android
 package.
 
+## AndroidIgnoredJavaDependency
+
+`<AndroidIgnoredJavaDependency>` is used in conjunction with [Java Dependency Resolution](../JavaDependencyVerification.md).
+
+It is used to specify a Java dependency that should be ignored. This can be
+used if a dependency will be fulfilled in a way that Java dependency resolution
+cannot detect.
+
+```xml
+<!-- Include format is {GroupId}:{ArtifactId} -->
+<ItemGroup>
+  <AndroidIgnoredJavaDependency Include="com.google.errorprone:error_prone_annotations" Version="2.15.0" />
+</ItemGroup>
+```
+See the [Java Dependency Resolution documentation](../JavaDependencyVerification.md)
+for more details.
+
+This build action was introduced in .NET 9.
```

Comment:
This section should explicitly call out supported item metadata, a'la https://github.com/xamarin/xamarin-android/blob/9f548d50cc03e24b85fe777a183251b184acdd93/Documentation/guides/building-apps/build-items.md#javasourcejar

> …the following MSBuild metadata is supported:
> 
> * `%(CopyrightFile)`: A path to a file that contains copyright
>     information for the Javadoc contents, which will be appended to
>     all imported documentation.
> 
> * `%(UrlPrefix)`: A URL prefix to support linking to online
>     documentation within imported documentation.
> 
> * `%(UrlStyle)`: The "style" of URLs to generate when linking to
>     online documentation.  Only one style is currently supported:
>     `developer.android.com/reference@2020-Nov`.